_includes/configuration.md

```diff
@@ -1,152 +1,47 @@
 
-1.  Add a disk to the infrastructure hosting your appliance.
+- Add a disk to the infrastructure that is hosting your appliance.
 
-2.  Configure the database.
-
-Configure the {{ site.data.product.title_short }} appliance using the internal
-appliance console.
+Configure the {{ site.data.product.title_short }} appliance by using the internal appliance console.
 
 ### Accessing the Appliance Console
 
-1.  Start the appliance and open a terminal console.
+1. Start the appliance and open a terminal console.
 
-2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
-    summary screen displays.
+2. Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance summary screen displays.
 
-3.  Press `Enter` to manually configure settings.
+3. Press `Enter` to manually configure settings.
 
-4.  Press the number for the item you want to change, and press `Enter`.
-    The options for your selection are displayed.
+4. Press the number for the item you want to change, and press `Enter`. The options for your selection are displayed.
 
-5.  Follow the prompts to make the changes.
+5. Follow the prompts to make the changes.
 
-6.  Press `Enter` to accept a setting where applicable.
+6. Press `Enter` to accept a setting where applicable.
 
 **Note:**
 
-The {{ site.data.product.title_short }} appliance console automatically logs out
-after five minutes of inactivity.
-
-### Configuring a Database
-
-{{ site.data.product.title_short }} uses a database to store information about the
-environment. Before using {{ site.data.product.title_short }}, configure the database
-options for it; {{ site.data.product.title_short }} provides the following two
-options for database configuration:
-
-  - Install an internal PostgreSQL database to the appliance
-
-  - Configure the appliance to use an external PostgreSQL database
-
-#### Configuring an Internal Database
-
-{% include configuration-db.md %}
-
-#### Configuring an External Database
-
-Based on your setup, you will choose to configure the appliance to use
-an external PostgreSQL database. For example, we can only have one
-database in a single region. However, a region can be segmented into
-multiple zones, such as database zone, user interface zone, and
-reporting zone, where each zone provides a specific function. The
-appliances in these zones must be configured to use an external
-database.
-
-The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
-specific settings for correct operation. For example, it must correctly
-reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. Due to these requirements, Red
-Hat recommends that external {{ site.data.product.title_short }} databases use a
```

Comment:
When keeping this section, we can drop the Red Hat bit.  I would just say "it is recommended that"